app/javascript/services/forest-data.js

```diff
@@ -27,6 +29,13 @@ const getLocationQuery = (country, region, subRegion) =>
     subRegion ? ` AND adm2 = ${subRegion}` : ''
   }`;
 
+const getRankingLocationQuery = (country, region, subRegion) =>
```

Comment:
I think we had a helper for this (getLocationQuery), is there a difference?

Reply:
oh! I didn't know it. Of course, I'm going to unify it.